README.md

```diff
@@ -6,6 +6,7 @@
 [![Total Downloads](https://poser.pugx.org/ttn/tea/downloads.svg)](https://packagist.org/packages/ttn/tea)
 [![Latest Unstable Version](https://poser.pugx.org/ttn/tea/v/unstable.svg)](https://packagist.org/packages/ttn/tea)
 [![License](https://poser.pugx.org/ttn/tea/license.svg)](https://packagist.org/packages/ttn/tea)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](CODE_OF_CONDUCT.md) 
```

Comment:
We're currently using version 1.4 of the code, but the badge is for version 2.0. So we either need to downgrade the badge to 1.4, or upgrade our text to version 2.0 first. Would you be willing to provide a PR for upgrading the text first?